### PR TITLE
appflowy: 0.0.6.2 -> 0.0.8

### PR DESCRIPTION
**Pull request by DarkOnion0**
###### Description of changes

Update appflowy to the [latest version](https://github.com/AppFlowy-IO/AppFlowy/releases/tag/0.0.8)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/office/appflowy/default.nix [hunk 1]
```diff
@@ -47,16 +47,18 @@ stdenv.mkDerivation rec {
...
+      binPath = lib.makeBinPath [
+        xdg-user-dirs
+      ];
+    in
+    ''
+      # Add missing libraries to appflowy using the ones it comes with
+      makeWrapper $out/opt/app_flowy $out/bin/appflowy \
+            --set LD_LIBRARY_PATH "$out/opt/lib/" \
+            --prefix PATH : "${binPath}"
+    '';
```
Comment (SuperSandro2000) [inline pkgs/applications/office/appflowy/default.nix:61]:
```suggestion
  preFixup = ''
    # Add missing libraries to appflowy using the ones it comes with
    makeWrapper $out/opt/app_flowy $out/bin/appflowy \
      --set LD_LIBRARY_PATH "$out/opt/lib/" \
      --prefix PATH : "${lib.makeBinPath [ xdg-user-dirs ]}"
  '';
```

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -186,7 +186,7 @@ with pkgs;
   } ../build-support/setup-hooks/auto-patchelf.sh;
 
   appflowy = callPackage ../applications/office/appflowy {
-    openssl = openssl_1_1;
+    openssl = openssl_3;
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:189]: Then we might as well remove this

Comment (DarkOnion0) [reply pkgs/top-level/all-packages.nix:189]: For sure :sweat_smile:

**Comment by deejayem** [comment]
Result of `nixpkgs-review pr 203979` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>appflowy</li>
  </ul>
</details>

**Review by deejayem — APPROVED** [review]
I was able to succesfully build and run this on x86_64-linux.

**Pull request merged**